lib/client-assets.php

```diff
@@ -218,116 +263,74 @@ function gutenberg_register_packages_scripts() {
 		);
 	}
 }
+add_action( 'wp_default_scripts', 'gutenberg_register_packages_scripts' );
 
 /**
- * Registers common scripts and styles to be used as dependencies of the editor
- * and plugins.
+ * Registers all the WordPress packages styles that are in the standardized
+ * `build/` location.
  *
- * @since 0.1.0
- */
-function gutenberg_register_scripts_and_styles() {
-	global $wp_scripts;
-
-	gutenberg_register_vendor_scripts();
-	gutenberg_register_packages_scripts();
-
-	// Add nonce middleware which accounts for the absence of the heartbeat
-	// listener. This relies on API Fetch implementation running middlewares in
-	// order of last added, and that the original nonce middleware would defer
-	// to an X-WP-Nonce header already being present. This inline script should
-	// be removed once the following Core ticket is resolved in assigning the
-	// nonce received from heartbeat to the created middleware.
-	//
-	// See: https://core.trac.wordpress.org/ticket/46107 .
-	// See: https://github.com/WordPress/gutenberg/pull/13451 .
-	if ( isset( $wp_scripts->registered['wp-api-fetch'] ) ) {
```

Comment:
@ellatrix - it looks like we can safely remove this override which add `wp-hooks` as a dependency since you landed: https://github.com/WordPress/gutenberg/pull/16683. Can you confirm?

Reply:
Yes, there is no more hooks dependency!